Makefile

```diff
@@ -529,7 +532,7 @@ IMAGE_REVIEWERS ?= $(shell ./hack/get-project-maintainers.sh)
 
 .PHONY: $(RELEASE_DIR)
 $(RELEASE_DIR):
-	mkdir -p $(RELEASE_DIR)/
+	mkdir -p $(RELEASE_DIR)/main
```

Comment:
nit: Hm. Should we really do it in this target? If we need it in release-flavors-main I would suggest to do it there

Reply:
I wonder if we can simply overwrite RELEASE_DIR in l. 295 to $(RELEASE_DIR)/main

Reply:
Although I'm not really sure why generate them into the RELEASE_DIR. Would it make more sense to directly generate them into the target?  ($(E2E_TEMPLATE_DIR)/main)

Reply:
I assume the "release-flavors-main" target is only necessary for the main branch and we do something else for older releases (?). Otherwise not sure how we can actually get "older" cluster templates with the flavor gen from main